pkg/api/handlers/compat/networks.go

```diff
@@ -74,6 +74,9 @@ func convertLibpodNetworktoDockerNetwork(runtime *libpod.Runtime, network *netty
 	for _, con := range cons {
 		data, err := con.Inspect(false)
 		if err != nil {
+			if errors.Is(err, define.ErrNoSuchCtr) {
```

Comment:
I think this must also ignore `ErrCtrRemoved`

Reply:
Done